ModelBundle/Repository/RepositoryTrait/KeywordableTrait.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\Repository\RepositoryTrait;
+
+use OpenOrchestra\ModelInterface\Repository\RepositoryTrait\KeywordableTraitInterface;
+/**
+ * Trait KeywordableTrait
+ */
+trait KeywordableTrait
```

Comment:
@itkg-nanne : A trait is used to add attributes and methods to access/set/play with those attributes. Here are described some kind of static functions without any relation on objects using the trait. So I think a better solution should be to use a standard Symfony service dedicated to this transformations. This service could be injected anywhere it is required.